Tests/Engine/Settings.tests.ps1

```diff
@@ -134,15 +134,15 @@ Describe "Settings Class" {
             }
 
             $settings = New-Object -TypeName $settingsTypeName  -ArgumentList $settingsHashtable
-            $settings.CustomRulePath.Count | Should Be $rulePaths.Count
-            0..($rulePaths.Count - 1) | ForEach-Object { $settings.CustomRulePath[$_] | Should be $rulePaths[$_] }
+            $settings.CustomRulePath.Count | Should -Be $rulePaths.Count
+            0..($rulePaths.Count - 1) | ForEach-Object { $settings.CustomRulePath[$_] | Should -Be $rulePaths[$_] }
```

Comment:
wouldn't this be more easily expressed as:
```
for($i = 0; $i -lt $rulePaths.Count; $i++ ) {
    $settings.CustomRulePath[$i] | Should -Be $rulePaths[$i]
}
```

Reply:
I find the current way actually quite neat and reasonably readable. If I had written it then it would look like something:
````powershell
foreach ($i in 0..($rulePaths.Count-1)) {
    $settings.CustomRulePath[$i] | Should -Be $rulePaths[$i]
}
````
At the end I don't mind too much but I would rather not touch existing tests too much if they are generally fine. But up to you whatever you prefer. If you want to make the style of all tests consistent, then this work should probably be a separate PR.